src/polyglot-notebooks-vscode-insiders/README.md

```diff
@@ -10,18 +10,18 @@ Polyglot Notebooks are fully interoperable with Jupyter and support the `.ipynb`
 
 The following languages are supported by Polyglot Notebooks:
 
-| Language                                    | Variable sharing supported |
-|---------------------------------------------|-------------------------------------------------------|
-| C#                                          |        ✅                 |
-| F#                                          |        ✅                 |   
-| PowerShell                                  |        ✅                 |          
-| JavaScript                                  |        ✅                 |          
-| SQL                                         |        ✅                 |   
-| KQL ([Kusto Query Language](https://learn.microsoft.com/en-us/azure/data-explorer/kusto/query/))    |        ✅       |       
-| HTML                                        |        ⛔                 |     
-| [Mermaid](https://mermaid.js.org/intro/)    |        ⛔                 |        
-| Python (Preview)                            |        ✅                 |          
-| R (Preview)                                 |        ✅                 |          
+| Language                      | Variable sharing |
+|-------------------------------|------------------|
+| C#                            |        ✅       |
+| F#                            |        ✅       |   
+| PowerShell                    |        ✅       |          
+| JavaScript                    |        ✅       |          
+| SQL                           |        ✅       |   
+| KQL ([Kusto Query Language](https://learn.microsoft.com/en-us/azure/data-explorer/kusto/query/))    |        ✅       |  
+| [Python](docs/jupyter-in-polyglot-notebooks.md)  |        ✅       |
```

Comment:
Should we be marking these as preview features still? I understand they are shipping inbox but we've added a preview tag on them in the product, so it would be nice for these to be consistent.

Reply:
There's no preview tag in the product for individual kernels.